src/sage/topology/simplicial_complex.py

```diff
@@ -1044,6 +1044,7 @@ def __init__(self,
             self._is_immutable = False
             if not is_mutable or is_immutable:
                 self.set_immutable()
+            self._bbn = copy(C._bbn)
```

Comment:
I don't think this needs to be copied as we never change the cache once it has been set.

Reply:
Agreed, I will remove that line.

Reply:
No, you misunderstood. The line should just be `self._bbn = C._bbn` without the copy.

Reply:
My apologies, I added it now.